java/core/src/java/org/apache/orc/impl/InStream.java

```diff
@@ -621,9 +631,20 @@ private ByteBuffer slice(int chunkLength) throws IOException {
           chunkLength + " bytes");
     }
 
-    void seek(long desired) throws IOException {
+    /**
+     * Seek to the desired chunk based on the input position.
+     *
+     * @param desired position in the compressed stream
+     * @return Indicates whether a seek was performed or not
+     * @throws IOException when seeking outside the stream bounds
+     */
+    boolean seek(long desired) throws IOException {
       if (desired == 0 && bytes == null) {
-        return;
+        return false;
```

Comment:
Lets remove ORC-758 leftovers from the PR

Reply:
Removed ORC-758 references